build-hyperv.ps1

```diff
@@ -72,10 +73,13 @@ Write-Output "External virtual switch detected - good."
 Test-CustomVersionOfVagrantDscPluginIsInstalled
 Test-PluginInstalled "vagrant-winrm-syncedfolders"
 
-Write-Output "Running Pester Tests"
-$result = Invoke-Pester -OutputFile PesterTestResults.xml -OutputFormat NUnitXml -PassThru
-if ($result.FailedCount -gt 0) {
-  exit 1
+if(-not $SkipPester)
```

Comment:
Confusing why this block is different to `build-aws.ps`?

Reply:
THis'll go into a common function as part of issue #183